src/components/main_container/MapView.tsx

```
@@ -0,0 +1,199 @@
+// Import the @terraformer/wkt package
+// Use the parse function from @terraformer/wkt to handle WKT strings
+import { wktToGeoJSON } from '@terraformer/wkt';
+import React, { useEffect, useRef } from 'react';
+import 'ol/ol.css';
+import { Map, View } from 'ol';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)